### DISCUSSION TITLE
Event shown as "Sold out" when only one product is sold out

### DISCUSSION THREAD

**Discussion by blutonium** [pretix/pretix#2579] [General]
We normally have multiple ticket types, including a very limited "Early Bird" one.
Once this is sold out, the whole event is marked as "Sold out" at the organizer front page, although there are still other products with quota available. 

There's no way we know to solve this problem besides deactivating the sold out ticket type, which is not wanted in most of the cases. We want to show that there was another, cheaper ticket type, which is already sold out.

**Comment by raphaelm** [comment]
Can you show screenshots of the organizer front page, event front page, and quota settings?

**Comment by blutonium** [comment]
Yes, of course:

![Screenshot from 2022-04-08 11-00-22](https://user-images.githubusercontent.com/30829664/162402753-6dd88664-aeb0-4163-8a66-16ce1df4fa63.png)
![Screenshot from 2022-04-08 11-00-15](https://user-images.githubusercontent.com/30829664/162402755-39c1bb06-0e90-4ba8-9dc6-6d08a029f88b.png)
![Screenshot from 2022-04-08 11-00-06](https://user-images.githubusercontent.com/30829664/162402756-dfbd013c-d256-47d7-aa98-556296257d0b.png)

**Reply by raphaelm** [reply]
Please have a look at the detail settings of the second quota, as the crossed-out :eye: icon indicates, you specifically excluded the quota from having an effect on the calendar

**Reply by blutonium** [reply]
Sorry for this, this really is exactly what we wanted to achieve. Thank you very much!
